torch/csrc/jit/passes/graph_rewrite_helper.cpp

```diff
@@ -34,7 +34,31 @@ c10::optional<IValue> getIValue(
   return toIValue(getValue(name, match_vmap, vmap));
 }
 
-void replaceConvolutionWithConv2d(std::shared_ptr<Graph>& graph) {
+void get_conv_params(
+    const Match& match,
+    const std::unordered_map<std::string, Value*>& vmap,
+    std::unordered_map<std::string, c10::IValue>& calc_values) {
```

Comment:
Also, why is `calc_values` passed as a reference? why not just return a new map?